packages/core/src/entities/feature.ts

```diff
@@ -263,7 +264,7 @@ export class Feature<
                     continue;
                 }
                 for (const key of Object.keys(featureOutput)) {
-                    settingUpFeature[key] = (featureOutput as any)[key];
+                    settingUpFeature[key] = (featureOutput)[key as keyof typeof featureOutput];
```

Comment:
Is casting necessary here?

Reply:
since Object.keys(featureOutput) returns string[], the casting is required. 
I think casting the key has more intent than casting as any